MapboxNavigation/MapboxSpeechController.swift

```diff
@@ -0,0 +1,149 @@
+import Foundation
+import MapboxSpeech
+import AVFoundation
+import MapboxCoreNavigation
+import CoreLocation
+import MapboxDirections
+
+/**
+ `PollyVoiceController` extends the default `RouteVoiceController` by providing support for AWSPolly. `RouteVoiceController` will be used as a fallback during poor network conditions.
+ */
+@objc(MBMapboxSpeechController)
+public class MapboxSpeechController: RouteVoiceController {
```

Comment:
MapboxSpeechController and RouteVoiceController have dissimilar names, so the relationship is unclear. Some possibilities:

* `MapboxVoiceController` inherits from `RouteVoiceController`
* `MapboxSpeechController` inherits from `RouteSpeechController`
* `RemoteVoiceController` inherits from `RouteVoiceController`